src/controllers/sessionLockController.ts

```diff
@@ -243,6 +256,54 @@ export default class SessionLockController {
       )
     );
 
+  /**
+   * Lock a user authentication and clear all its session data
```

Comment:
```suggestion
   * Release a lock previously set by the user
```

Reply:
Done in [644e3f7](https://github.com/pagopa/io-backend/pull/1065/commits/644e3f7cd999412b3beb1a5af97653cb120d6ecd)